src/util/metrics.ts

```diff
@@ -26,32 +78,179 @@ export default class Metrics_App {
 	constructor({ host }: IAppConfiguration) {
 		const args = parseArgs();
 
+		this.includeQueryParams = Boolean(args.prometheus_queryparams);
```

Comment:
Generally speaking, I like to stay away from using `Boolean()`, `Number()` etc because its a bit underperforming. I think here you can just do `!!args.prometheus_queryparams`